perception/object_merger/src/object_association_merger/node.cpp

```diff
@@ -69,9 +69,10 @@ ObjectAssociationMergerNode::ObjectAssociationMergerNode(const rclcpp::NodeOptio
   remove_overlapped_unknown_objects_ =
     declare_parameter<bool>("remove_overlapped_unknown_objects", true);
   overlapped_judge_param_.precision_threshold =
-    declare_parameter<double>("precision_threshold_to_judge_overlapped", 0.4);
+    declare_parameter<double>("precision_threshold_to_judge_overlapped");
```

Comment:
https://github.com/autowarefoundation/autoware.universe/pull/1902/files#r973732847
same comment

Reply:
I added distance_threshold_list to  overlapped_judge_param_ struct.